.circleci/config.yml

```diff
@@ -165,7 +222,71 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
-            - .
+            - ./packages
+            - ./dist
+
+      - store_artifacts:
+          path: /root/repo/dist/
+
+      - store_artifacts:
+          path: /root/repo/pyodide-build.tar.gz
+
+      - store_artifacts:
+          path: /root/repo/packages/build-logs
+
+      - store_artifacts:
+          path: /root/repo/build-logs.tar.gz
+
+  build-packages-opencv-python:
```

Comment:
There is getting to be a lot of repetition in these package build steps, maybe as a followup we could try to extract some of it into custom-defined steps or something.

Reply:
Done :)
(Duplicated `post_steps` looks little ugly, but I can't find a way to pass a list parameter)